plugins/lookup/laps.py

```diff
@@ -0,0 +1,233 @@
+# Copyright: (c) 2024, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r"""
+name: laps
+author: Nico Ohnezat (@no-12)
+short_description: Inventory plugin for Active Directory
+version_added: 2.2.0
+description:
+- Lookup plugin that retrieves the LAPS password information for multiple hosts from an Active Directory server.
+options:
+  _terms:
+    description:
+    - One or multiple C(CommonName) of the computer objects to search for.
```

Comment:
I think using the `sAMAccountName` might make more sense for computer accounts. Most people will know it by that value rather than the LDAP cn. We would have to ensure we add the `$` suffix in the code if the user didn't supply it though.